qoqo/build.rs

```diff
@@ -161,6 +168,184 @@ const SOURCE_FILES: &[&str] = &[
     "src/operations/analog_operations.rs",
 ];
 
+#[cfg(feature = "doc_generator")]
+fn str_to_type(res: &str) -> Option<String> {
+    match res {
+        s if s.contains("Pragma") => Some("Operation".to_owned()),
+        "CalculatorFloat" => Some("Tuple[float, str]".to_owned()),
```

Comment:
Shouldn't this just be either `CalculatorFloat`, as the type exists in python too, or `Union` instead of `Tuple`?

Reply:
I changed to Union because withou have qoqo_calcultor with a fonctionning docstring CalculatorFloar won't be recognize